systemtests/src/main/java/io/enmasse/systemtest/utils/IoTUtils.java

```diff
@@ -177,7 +177,15 @@ public static void waitForIoTProjectReady(Kubernetes kubernetes, IoTProject proj
         assertThat(project.getStatus().getDownstreamEndpoint().getPort(), not(is(0)));
     }
 
-    private static void waitForIoTProjectDeleted(Kubernetes kubernetes, IoTProject project) throws Exception {
+    public static boolean isIoTInstalled(Kubernetes kubernetes) {
+        return kubernetes.getCRD(IoTCrd.project().getMetadata().getName()) != null;
```

Comment:
This will always be true, the CRD is always created when installing.

Reply:
Indeed. That came back by accident. It is removed again.